src/components/footer.tsx

```diff
@@ -0,0 +1,60 @@
+import { useContext } from 'react';
+import { TodosContext, EStatus, filterTodos } from '../TodosContext';
+
+export const TodosFooter = () => {
+  const { todos } = useContext(TodosContext);
+  const { setFilterType, filterType, setTodos } = useContext(TodosContext);
+
+  if (todos.length > 0) {
```

Comment:
better to move this logic to TodoApp instead of the footer

the footer component's goal is to render the footer information, that component doesn't need to know if it should be rendered, or shouldn't